src/argilla/client/feedback.py

```diff
@@ -91,16 +90,7 @@ def responses_must_be_a_list(cls, v):
         return v
 
     class Config:
-        extra = Extra.forbid
-
-
-class OfflineFeedbackRecord(BaseModel):
```

Comment:
Do all Pydantic models doo validation during attribute assignment?

Reply:
Yes they do, unless you call `.construct()`, that just builds it as a `pydantic.BaseModel` ignoring the nested `pydantic.BaseModel`s and assuming those are Python dicts, which is not what we want here. So instantiating or calling `parse_obj` over a `pydantic.BaseModel` does validate it 😄